### PR TITLE
AAV-158 Canvas playground

### PR DESCRIPTION
**Pull request by Onatolich**

### CODE REVIEW

**Comment by Onatolich** [comment]
@itdev React as any other framework has its best practices. One of them is telling us that if component don't need to have a state, rewrite lifecycle methods or handle events we should arrange it as functional stateless component.

File: playground/components/Playground/Playground.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  position: fixed;
+  display: flex;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+
+  * {
+    box-sizing: border-box;
+  }
```
Comment (itdev) [inline playground/components/Playground/Playground.css]: Do you really need set `box-sizing` for all elements?

Comment (Onatolich) [reply playground/components/Playground/Playground.css]: Yes

Comment (Onatolich) [reply playground/components/Playground/Playground.css]: It won't affect Canvas styles as there is no such thing as box-sizing in svg.

**Pull request merged**